### PR TITLE
Profile: convert data download to React

### PR DESCRIPTION
**Pull request by simison**
#### Proposed Changes

* Another quick Angular->React conversion of tiny bit of account page. Gets these strings to translation.

I left out the analytics bits because they aren't important.

#### Testing Instructions

* Open http://localhost:3000/profile/edit/account
* See the "download" bit, each link should work.
Screenshot 2020-12-24 at 00 32 55

### CODE REVIEW

File: modules/users/client/views/profile/profile-edit-account.client.view.html [hunk 1]
```diff
@@ -1,8 +1,6 @@
 <!-- Email -->
 <div class="panel panel-default">
-  <div class="panel-heading">
-    Email
-  </div>
+  <div class="panel-heading">Email</div>
```
Comment (simison) [inline modules/users/client/views/profile/profile-edit-account.client.view.html:3]: These were just some changes done by Prettier/Eslint.

**Review by hlibbabii — APPROVED** [review]
👍🏼

**Comment by hlibbabii** [comment]
Do we need to add a link for experiences, btw?

**Comment by simison** [comment]
Yeah, it would be good to add experiences here once it's live or earlier behind the feature flag. Part of the reason I already looked at this now.

**Pull request merged**